src/ol/source/imagevectorsource.js

```diff
@@ -58,6 +58,12 @@ ol.source.ImageVector = function(options) {
 
   /**
    * @private
+   * @type {number}
+   */
+  this.renderBuffer_ = options.renderBuffer || 100;
```

Comment:
I don't know if it's a realistic use case but this prevent to have a `0` renderBuffer